Separa el front-end de la API

devarops opened the pull request:
Dividí el repo **tablero** en dos repos:

- [tablero_front](http://islasgeci.org:5000/)
- [tableto_api](http://islasgeci.org:500/api/v1/dashboard)

Estos son los cambios mínimos necesarios para lograr la separación.

devarops committed:
Crea primer versión de la API

devarops committed:
Elimina lo que ahora es responsabilidad de la API

devarops committed:
Agrega nuevo requerimiento

devarops committed:
Importa función get_dashboard()

devarops committed:
Regresa tablero como pandas

devarops committed:
Usa nombre de los campos en lugar de índices

devarops committed:
Usa dirección publica en internet

analislas committed:
Elimina API que ahora es parte del otro repo tablero_api

analislas committed:
Renombra imagen a islasgeci/tablero_front

analislas committed:
Corrige URL de API ya que le faltaba el '.org'

efren-cabrera commented on tablero/Dashboard.py:
```diff
@@ -42,4 +42,4 @@ def get_dashboard():
         renglon_concatenar = {"repo": repo, "objetivo": objetivo,
```
Estas funciones no se usan en el front-end, deberías eliminar este código

efren-cabrera commented on requirements.txt:
```diff
@@ -1,3 +1,4 @@
 flask
 numpy
 pandas
```
Aquí no ocupas `pandas` ni `numpy`

devarops committed:
Actualiza el contenido cada 100 segundos

devarops committed:
Usa jQuery para dejar de usar Flask y poder ponerlo en islas.org.mx

analislas committed:
Usa imagen de Apache en lugar de Python

devarops committed:
Elimina archivos innecesarios en el front end

Estos archivos estaban aquí porque antes el front end y el back end estaban en el mismo repositorio

devarops committed:
Ignora todos los archivos cuyos nombres unicien con punto

efren-cabrera commented on templates/index.html:
```diff
@@ -1,31 +1,42 @@
-            {% endfor %}
         </table>
     </div>
+    <script>
```
¿Qué te parece si ponemos esto en un archivo separado? Así no mezclamos la interfaz (html) con el código

devarops replied:
Me parece buena idea. Así lo haré.

efren-cabrera reviewed (changes requested):
¿Esto ya está corriendo en nuestro servidor?

devarops commented:
> ¿Esto ya está corriendo en nuestro servidor?

Sí, esta consignación es la que está corriendo.

devarops committed:
Renombra templates/ a public_html/ y pon JavaScript en otro archivo

devarops committed:
Cambia título a "Tablero de Reproducibilidad"

devarops committed:
Escribe símbolo con código HTML

devarops committed:
Carga la página cada 10 minutos

efren-cabrera committed:
Da formato a documento

efren-cabrera committed:
Espera hasta que cargue la página para ejecutar código
